scripts/build-arch.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -eu
+
+export PKG_CONFIG_SYSROOT_DIR=/usr/aarch64-linux-gnu
```

Comment:
Looks aarch64 specific and `pkg-config-aarch64-linux-gnu` related, do we want to keep it?

Reply:
You mean the package that does not exist any more? I can try to remove the variable, yes.

Reply:
It does not build aarch64. Reverted.